ui/app/AppLayouts/Communities/panels/SharedAddressesSigningPanel.qml

```diff
@@ -61,43 +82,41 @@ ColumnLayout {
 
         spacing: Style.current.padding
 
+        StatusBaseText {
+            Layout.preferredWidth: parent.width
+            elide: Text.ElideRight
+            font.pixelSize: Constants.keycard.general.fontSize2
+            text: qsTr("To share %n address(es) with <b>%1</b>, authenticate the associated keypairs...", "", d.selectedSharedAddressesCount).arg(root.communityName)
```

Comment:
```suggestion
            text: qsTr("To share %n address(s) with <b>%1</b>, authenticate the associated keypairs...", "", d.selectedSharedAddressesCount).arg(root.communityName)
```

Reply:
This still needs fixing @saledjenic

Reply:
hahahah you really made me search for that pattern to finally change all occurrences, initially I changed only those you've commented ;)